kiva/compat.py

```diff
@@ -0,0 +1,23 @@
+"""
+Compatibility module to help support various versions of PIL and
+Pillow.
+
+"""
+import PIL
+from PIL import Image
+
+PILLOW_VERSION = getattr(PIL, 'PILLOW_VERSION', PIL.VERSION)
+
+
+def piltostring(image):
+    if PILLOW_VERSION.startswith('3'):
+        return image.tobytes()
```

Comment:
Actually, I think I still have a preference for checking for `tobytes()` instead of the version string. When Pillow goes to `4.0`, we still want this to work.